browser/scripts/node.js

```diff
@@ -239,6 +259,29 @@ Node.prototype.remove_slot = function(slot_type, suid) {
 	this.emit('slotRemoved', slot)
 }
 
+Node.prototype.setInputSlotValue = function(name, value) {
+	var slot = this.findInputSlotByName(name)
+
+	if (value === slot.def) {
+		delete this.uiSlotValues[name]
+		delete this.plugin.inputValues[name]
```

Comment:
i thought about that, but versioned engine already solves _most of_ that problem.